sources/packages/backend/workflow/test/2021-2022/parttime-assessment/awards-amounts/parttime-assessment-2021-2022-awards-amount-SBSD.e2e-spec.ts

```diff
@@ -0,0 +1,93 @@
+import { PROGRAM_YEAR } from "../../constants/program-year.constants";
```

Comment:
If we all devs agree, we must remove the program year from the file names as per https://github.com/bcgov/SIMS/pull/2923#discussion_r1513553502